### DISCUSSION TITLE
Can not update MCU

### DISCUSSION THREAD

**Discussion by GNiessen** [Guilouz/Klipper-Flsun-Speeder-Pad#35] [General]
I am getting an error:
Can not update MCU 'mcu' config as it is shutdown
Once the underlying issue is corrected, use the
"FIRMWARE_RESTART" command to reset the firmware, reload the
config, restart the host software.
Error configuring printer

I have tried everything I can see to do through the display to restart.  But the error does not go away.  Any help would be appreciated.

**Comment by Guilouz** [comment]
Many possible causes :
 - You have compiled firmware with bad parameters
 - You don't have retrieve your USB serial and not reported in your printer.cfg
 - You use a bad USB cable

**Reply by cyb1337** [reply]
What do you mean with that exactly? I have a similar issue, the guide refers to updating the MCU via microsd card but even after configuring the firmware with the correct parameters and also formatting the sd card as specified the printer does not install it and also does not rename the file.
